.gitignore

```diff
@@ -11,5 +11,6 @@
 /log/*.log
 /pkg/
 /spec/reports/
+.idea/*
```

Comment:
Did you mean to include this?  Seems like it could go into your local `.gitignore_global` file

Reply:
Removed